lib/iris/tests/results/netcdf/netcdf_save_hybrid_height.cdl

```diff
@@ -6,6 +6,7 @@ dimensions:
 	model_level_number = 10 ;
 variables:
 	float air_potential_temperature(time, model_level_number, grid_latitude, grid_longitude) ;
+		air_potential_temperature:_FillValue = -1.073742e+09f ;
```

Comment:
This relates to your above comment where a biggus array always has a fill value.